features/mbedtls/platform/COMPONENT_PSA_SRV_IMPL/src/default_random_seed.cpp

```diff
@@ -0,0 +1,19 @@
+
+#include "mbed.h"
+#include "default_random_seed.h"
+#include "psa_prot_internal_storage.h"
+
+int mbed_default_seed_read( unsigned char *buf, size_t buf_len )
```

Comment:
Style: Use Mbed OS style consistently. We seem to be mixing Mbed TLS and Mbed OS style in this file.